_posts/2022-03-02-getting-started-project-flotta.markdown

```diff
@@ -75,4 +93,14 @@ edgedeployments.management.project-flotta.io   2022-03-01T14:07:17Z
 edgedevices.management.project-flotta.io       2022-03-01T14:07:17Z
 ```
 
+By default, both operator manifests (for OCP and k8s) assume [ObjectBucketClaim CRD](https://github.com/kube-object-storage/lib-bucket-provisioner/blob/master/pkg/apis/objectbucket.io/v1alpha1/objectbucketclaim_types.go) is installed on the cluster.
+If it is not installed, there are two options, based on the needs:
+
+#### Disable Object Bucket Claim auto-creation
+To disable _ObjectBucketClaim_ auto-creation, Flotta's operator config map needs to be patched and Flotta's operator needs to be restarted:
```

Comment:
In this guide we use "`flotta` operator" instead of "Flotta's operator". Personally, I prefer "Flotta's operator" but I think we should be consistent and choose one of the options

Reply:
+1, changed all cases to Flotta